cli/cmd/plugin/unmanaged-cluster/test/e2e/utils/installTCE.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+set -x
+
+wget  https://github.com/vmware-tanzu/community-edition/releases/download/v0.11.0/tce-darwin-amd64-v0.11.0.tar.gz
```

Comment:
Shouldn't this test be installing the latest code, not the last full release?

Reply:
Hey @stmcginnis Thanks for reviewing. This is still in WIP.

Reply:
+1

Reply:
Going to change this to draft until we are ready.

Reply:
Hey @dvonthenen this is a old conversation if you had converted it as a draft after reading

> Hey @stmcginnis Thanks for reviewing. This is still in WIP.

Now I think I have covered all the reviews comments by everyone.
Thanks!